packages/components/src/Form/Inputs/InputTime/InputTime.tsx

```diff
@@ -533,9 +547,13 @@ const InputTimeInternal = forwardRef(
 const InputTimeLayout = styled.div`
   display: grid;
   grid-gap: 0.15rem;
-  grid-template-columns: repeat(4, min-content);
+  grid-template-columns: repeat(6, min-content);
+  grid-template-areas: 'hour colon minute 12 . warning';
```

Comment:
Because the columns are all sized to `min-content`, the grid doesn't know _which_ column should stretch to fill the available space. By manually typing out the `auto` size and making the last one `1fr` to fill the available space, the layout seems to work as we'd want:

```suggestion
  grid-template-columns: auto auto auto auto 1fr;
```

I think you should be safe to delete all the `grid-column` stuff too